packages/functions/src/api/service.ts

```diff
@@ -119,7 +127,7 @@ export class Service implements FirebaseFunctions, FirebaseService {
    * @param name The name of the trigger.
    */
   httpsCallable(name: string, options?: HttpsCallableOptions): HttpsCallable {
-    let callable = <HttpsCallable>(data?: any) => {
+    const callable = (data?: {}): Promise<HttpsCallableResult> => {
```

Comment:
`data?:` should be `{} | null`, not `{}`. I'm surprised this works, since there's a test that calls it with `null`.

Reply:
Gotcha. It works because `HttpsCallable` is defined in `functions-types` which still has `any` type. :( 
I will update `functions-types` as well.